<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>636</width>
    <height>410</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>10</y>
      <width>631</width>
      <height>341</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QGroupBox" name="groupBox">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>150</height>
        </size>
       </property>
       <property name="title">
        <string>TableView表格</string>
       </property>
       <widget class="QTableView" name="tableView">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>20</y>
          <width>601</width>
          <height>121</height>
         </rect>
        </property>
       </widget>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_2">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>150</height>
        </size>
       </property>
       <property name="title">
        <string>PlainTextEdit文本框</string>
       </property>
       <widget class="QPlainTextEdit" name="plainTextEdit">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>20</y>
          <width>601</width>
          <height>111</height>
         </rect>
        </property>
       </widget>
       <zorder>plainTextEdit</zorder>
       <zorder>groupBox</zorder>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QPushButton" name="pushButton">
         <property name="text">
          <string>表格居中对齐</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_2">
         <property name="text">
          <string>表格左对齐</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_3">
         <property name="text">
          <string>表格右对齐</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_4">
         <property name="text">
          <string>字体加粗</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionOpen"/>
   <addaction name="actionSave"/>
   <addaction name="actionView"/>
   <addaction name="separator"/>
   <addaction name="actionAppend"/>
   <addaction name="actionInsert"/>
   <addaction name="actionDelete"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionOpen">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/1/open.png</normaloff>:/1/open.png</iconset>
   </property>
   <property name="text">
    <string>打开文件</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/1/save.png</normaloff>:/1/save.png</iconset>
   </property>
   <property name="text">
    <string>保存文件</string>
   </property>
  </action>
  <action name="actionView">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/1/preview.png</normaloff>:/1/preview.png</iconset>
   </property>
   <property name="text">
    <string>预览</string>
   </property>
  </action>
  <action name="actionAppend">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/1/add.png</normaloff>:/1/add.png</iconset>
   </property>
   <property name="text">
    <string>添加行</string>
   </property>
  </action>
  <action name="actionInsert">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/1/insert.png</normaloff>:/1/insert.png</iconset>
   </property>
   <property name="text">
    <string>插入行</string>
   </property>
  </action>
  <action name="actionDelete">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/1/delete.png</normaloff>:/1/delete.png</iconset>
   </property>
   <property name="text">
    <string>删除行</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections/>
</ui>
